atom/browser/osr/osr_render_widget_host_view.h

```diff
@@ -184,6 +184,7 @@ class OffScreenRenderWidgetHostView
   std::unique_ptr<ui::CompositorLock> GetCompositorLock(
       ui::CompositorLockClient* client) override;
   void CompositorResizeLockEnded() override;
+  bool IsAutoResizeEnabled() const override;
```

Comment:
Was this not needed on non-macOS platforms?

Reply:
It's _only_ needed on non-macOS platforms, as far as I can tell. On macOS, it doesn't override anything in the parent class and we never call it.